tests/data_deep.py

```diff
@@ -149,31 +132,31 @@
     'List movies directed by Spielberg.':
     I([
         T(M(), R('instance of'), R('movie')),
-        T(M(), R('director'), R('Spielberg'))
+        T(M(), Li([R('director'), R('directed by')]), R('Spielberg'))
     ]),
 
     'Which books were authored by Victor Hugo?':
     I([
         T(M(), R('instance of'), R('book')),
-        T(M(), R('author'), R('Victor Hugo'))
+        T(M(), R('authored by'), R('Victor Hugo'), R('author'))
```

Comment:
It looks like there is an issue with the reverse predicate. We usually states that (Le Petit Prince, author, Saint-Exupéry) and it's what is done in other tests like "Who wrote \"Le Petit Prince\" and \"Vol de Nuit\"".

But this issue should not block the merge of this pull request.